pennylane/data/attributes/operator/operator.py

```diff
@@ -170,8 +170,6 @@ def consumes_types(cls) -> FrozenSet[Type[Operator]]:
                 qml.CatState,
                 qml.NumberOperator,
                 qml.TensorN,
-                qml.X,
-                qml.P,
```

Comment:
Looks like `qml.QuadX` and `qml.QuadP` aren't actually in this list. Do we need to add them in?

Reply:
Ah, good point, thanks!